src/Sarif.FunctionalTests/ConverterTestData/PREfast/AA_IndexAlias.cpp.xml.sarif

```diff
@@ -43,10 +43,10 @@
                     "startLine":5
                   }
                 },
-                "id":"1",
+                "step":0,
```

Comment:
* _Re_ `"id"`: I don't think so. PREfast's `KEYEVENT/ID` corresponds directly to SARIF's `annotatedCodeLocation.step`, and I think it's entirely within the spirit of the spec's guidelines for converters to translate the one to the other.

* _Re_: `"importance"`: I think the code is right as it stands. PREfast's `KEYEVENT/IMPORTANCE` property (whose value is either `Essential` or `Full`) corresponds directly to SARIF's `annotatedCodeLocation.importance (with the mappings `Essential => essential` and `Full => normal`).